ramls/finance.raml

```diff
@@ -0,0 +1,60 @@
+#%RAML 1.0
+
+title: Finance fund
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: Finance API
+    content: This documents the API calls that can be made to manage finances
+
+types:
+  finance: !include finance.json
+  financeCollection: !include financeCollection.json
+  
+  errors: !include raml-util/schemas/errors.schema
+  UUID:
+    type: string
+    pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+  language: !include raml-util/traits/language.raml
+  pageable:  !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/finance/funds:
```

Comment:
Please, specify endpoint path like
```
/finance:
  /funds:
```
At least it will simplify naming for generated interface classes. This way generated classes will be like Funds, Ledgers instead of FinanceFunds, FinanceLedgers etc.